proposals/instructor-training-selection.md

```diff
@@ -0,0 +1,175 @@
+at people who do not have such affiliation
+and hence have practically no alternative for getting trained as an instructor.
+
+## Eligibility
+
+### Groups
+
+-   With 4-6 people.
+-   Co-locate in one room for the two days
+-   Commit to jointly running one workshop no later than April 1st, 2016.
+
+### Members
+
+-   Be on the current waiting list
+    or
+    demonstrate sufficient competence with core tools used in the lessons they will teach.
+
+-   Watch a short video of someone teaching part of our core lessons and
+    write a summary of what was taught and how. Participants in the Fall
+    2015 pilot will be required to complete this step, and we will
+    select segments from the videos from SciPy 2015.
+
+## Application
+
+Groups **must** send a cover letter to [ADDRESS TO SEND COVER LETTER]()
+to apply for the two-day training course in December 2015.
+There is a cover letter sample at the end of this document.
+
+## Selection Criteria
+
+Groups will be selected based on:
+
+-   Previous contribution to SCF/DC,
+    e.g. members were learners, helpers or instructors in a workshop,
+    members open issues or pull request to one of SCF/DC lessons.
+
+-   Previous contribution to one group that organizes Software Carpentry-like workshops,
+    e.g. [The Hacker Within](https://thehackerwithin.github.com/),
+    Python and R local user groups.
+
+-   Plan for the local organization of the Instructor Training.
+
+-   Plan for the upcoming workshop.
+
+Groups will not be selected based on:
+
+-   Affiliation,
+    i.e. the affiliation with a particular institution, lab or university
+    will not be taken into account in selecting eligible groups.
+    That is, all institutions, labs and universities are treated equally.
+
+-   Academic qualification,
+    i.e. one group of professors will have the same opportunity
+    as one group of undergraduate students.
+
+The criteria for tie will be:
+
+-   timezones,
+    because instructor trainers and instructor in-training
+    should not be required to wake up too early or stay up too late.
+
+-   small number of instructors at city/province/state/country,
+    because SCF/DC believes that increasing the numbers of instructors
+    at one region will help increase the number of workshops
+    at that region and with that achieve their goal.
+
+## Selection
+
+Each members of
+
+-   the mentoring subcommittee,
+-   SCF steering committee and
+-   DC steering committee
+
+will have one vote for each group slot for the Instructor Training round.
+
+Members of more than one committee
+**will only have one vote per group slot**.
+
+For example,
+there will be eight slots
+and the committees composition are:
+
+-   Mentoring subcommittee: M1, M2, M3, M4, M5, M6, M7, M8.
+-   SCF steering committee: S1, S2, S3, S4.
+-   DC steering committee: D1, D2, D3, D4.
+
+So,
+M1, M2, M3, M4, M5, M6, M7, M8,
+S1, S2, S3, S4,
+D1, D2, D3, and D4
+will have eight votes each.
+
+## Group Confirmation Process
```

Comment:
Rather than saying we'll email people, say that selected groups will have one week to confirm participation.  If they fail to do so, their slot will be given to the next group in sequence.